tests/chart_tests/test_git_sync_relay_deployment.py

```diff
@@ -121,3 +121,60 @@ def test_gsr_deployment_with_ssh_credentials_and_known_hosts(self, kube_version)
             "--wait=333",
             "--root=/git",
         ]
+
+    def test_gsr_deployment_without_ssh_credentials_and_known_hosts(self, kube_version):
+        """Test that a valid deployment is rendered when enabling git-sync with ssh credentials and known hosts and other custom configs."""
```

Comment:
I don't think this test is doing what we need it to do. EG: it doesn't test that there is no secret.

Reply:
Wait, I see that it actually is checking that now.